apps/aehttp/test/aehttp_integration_SUITE.erl

```diff
@@ -1309,6 +1323,34 @@ block_txs_list_by_hash_invalid_range(_Config) ->
         ]),
     ok.
 
+%% ============================================================
+%% Websocket tests 
+%% ============================================================
+
+ws_get_genesis(_Config) ->
+    {Host, Port} = ws_host_and_port(),
+    {ok, ConnPid} = ?WS:start(Host, Port),
+    ?WS:send(ConnPid, chain, get, #{height => 0}),
+    ok = ?WS:subscribe_for_event(ConnPid, chain, requested_data),
+    {ok, Payload} = ?WS:wait_for_event(chain, requested_data),
+    {ok, Block} = maps:find(<<"block">>, Payload),
+    {ok, 200, BlockMap} = get_internal_block_by_height(0, false),
+    ExpectedBlockMap = maps:remove(<<"data_schema">>, BlockMap),
+    Block = ExpectedBlockMap,
+    ok = ?WS:unsubscribe_for_event(ConnPid, chain, requested_data),
+    aehttp_ws_test_utils:stop(ConnPid),
+    ok.
+
+ws_block_mined(_Config) ->
+    {Host, Port} = ws_host_and_port(),
+    {ok, ConnPid} = ?WS:start(Host, Port),
+    ok = ?WS:subscribe_for_event(ConnPid, miner, mined_block),
```

Comment:
Could you please ensure this is [documented](https://github.com/aeternity/protocol/blob/68b27350f157e05c6930d037a626695e6a4fad3b/epoch/api/README.md#websocket-api-definition) ?
(It may require moving out of the oracle page some more generic portions.)